implementations/micrometer-registry-statsd/src/test/java/io/micrometer/statsd/StatsdMeterRegistryPublishTest.java

```diff
@@ -329,6 +329,45 @@ void receiveParallelMetricsSuccessfully(StatsdProtocol protocol) throws Interrup
         server.disposeNow();
     }
 
+    @Test
+    void counterIncrementDoesNotWorkWithoutSleepAfterRegistryStart() throws InterruptedException {
```

Comment:
Are the names of the tests switched? It looks like this is showing it works with sleeping.